src/index.js

```diff
@@ -483,21 +487,58 @@ class JanusAdapter {
   }
 
   async createSubscriber(occupantId) {
+    if (this.leftOccupants.has(occupantId)) {
+      console.warn(occupantId + ": cancelled occupant connection, occupant left before subscription negotation.");
+      return null;
+    }
```

Comment:
it's actually a longer period than that -- we serially go through each of the initial occupants and set up the peer connections. (janus didn't like it when we tried to do this in parallel and would fail randomly.) so if there are 5 occupants, by the time you get to negotiating the 5th one a significant amount of time has passed (5-15 seconds potentially) so if that person leaves within that window the negotation will fail.